.github/ISSUE_TEMPLATE.md

```diff
@@ -17,8 +17,8 @@
 
 <!-- Example
 1. Put a `paper-foo` element in the page.
-2. Open the page in a web browser.
-3. Click the `paper-foo` element.
+* Open the page in a web browser.
```

Comment:
This was an odd change: now it's a mix of a numbered and a bulleted list? I've used
```
1. Item
1. Item
1. Item
```
to get:

1. Item
1. Item
1. Item